### PR TITLE
Preference warning fix

### PR DESCRIPTION
**Pull request by osodebailar**
Fix to eliminate the following warning , seen in logcat, when open preference dialog:
 
info.nightscout.androidaps E/PreferenceGroup: PreferenceCategory should have a key defined if it contains an expandable preference

### CODE REVIEW

File: app/src/main/res/xml/pref_automation.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/automation"
```
Comment (philou78) [inline app/src/main/res/xml/pref_automation.xml:6]: android:key shouldn't be "@string/key_..." ?

File: app/src/main/res/xml/pref_datachoices.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/data_choices"
```
Comment (philou78) [inline app/src/main/res/xml/pref_datachoices.xml:6]: android:key shouldn't be "@string/key_data_choices_settings" ?

File: app/src/main/res/xml/pref_openhumans.xml [hunk 3]
```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:validate="http://schemas.android.com/apk/res-auto">
 
-    <PreferenceCategory android:title="@string/open_humans">
+    <PreferenceCategory
+        android:key="open_humans"
```
Comment (philou78) [inline app/src/main/res/xml/pref_openhumans.xml:7]: android:key shouldn't be "@string/key_..." ?

File: app/src/main/res/xml/pref_tidepool.xml [hunk 4]
```diff
@@ -4,6 +4,7 @@
     xmlns:validate="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/tidepool"
```
Comment (philou78) [inline app/src/main/res/xml/pref_tidepool.xml:7]: android:key shouldn't be "@string/key_tidepool" or "@string/key_tidepool_settings" ?

File: danar/src/main/res/xml/pref_danar.xml [hunk 5]
```diff
@@ -4,6 +4,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/danar_pump_settings"
```
Comment (philou78) [inline danar/src/main/res/xml/pref_danar.xml:7]: android:key shouldn't be "@string/key_..." ?

File: medtronic/src/main/res/xml/pref_medtronic.xml [hunk 6]
```diff
@@ -4,6 +4,7 @@
     xmlns:validate="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/medtronic_name"
```
Comment (philou78) [inline medtronic/src/main/res/xml/pref_medtronic.xml:7]: android:key shouldn't be "@string/key_medtronic_settings" ?

File: app/src/main/res/xml/pref_automation.xml (hunk 1 above)
Comment (andyrozman) [reply app/src/main/res/xml/pref_automation.xml:6]: It should be key_automation... All keys are defined as untranslatable string resources

**Comment by osodebailar** [comment]
not needed anymore @MilosKozak  ??

**Pull request closed without merging**